src/ol3cesium.js

```diff
@@ -117,14 +117,15 @@ olcs.OLCesium = function(options) {
    */
   this.hiddenRootGroup_ = null;
 
+  var sceneProperties = options.sceneProperties !== undefined ? options.sceneProperties : {};
```

Comment:
The variable must be typed with the typedef to be created in the Cesium.externs.js file:
```
/**
  @type {Cesium.SceneOptions}
 */
```